app/src/main/java/com/vitorpamplona/amethyst/ui/MainActivity.kt

```diff
@@ -22,7 +22,7 @@ import com.vitorpamplona.amethyst.ui.screen.AccountScreen
 import com.vitorpamplona.amethyst.ui.screen.AccountStateViewModel
 import com.vitorpamplona.amethyst.ui.theme.AmethystTheme
 
-class MainActivity : ComponentActivity() {
+class MainActivity : FragmentActivity() {
```

Comment:
Biometrics are *only* available in FragmentActivity. Seems to be a hard requirement, no response from Google on issues open for more than 2 years already.